tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/AspNetCore5IastTests.cs

```diff
@@ -152,6 +154,7 @@ public abstract class AspNetCore5IastTests : AspNetBase, IClassFixture<AspNetCor
         protected static readonly Regex LocationMsgRegex = new(@"(\S)*""location"": {(\r|\n){1,2}(.*(\r|\n){1,2}){0,3}(\s)*},");
         protected static readonly Regex ClientIp = new(@"["" ""]*http.client_ip: .*,(\r|\n){1,2}");
         protected static readonly Regex NetworkClientIp = new(@"["" ""]*network.client.ip: .*,(\r|\n){1,2}");
+        protected static readonly Regex HashRegex = new(@"(\S)*""hash"": (-){0,1}([0-9]){1,12},(\r|\n){1,2}      ");
```

Comment:
nit: you could probably move these helper regexes to a separate class instead of copy-pasting them in 3 places. e.g. you could create: 

```csharp
public static class IastVerifyScrubberExtensions
{
    private static readonly Regex LocationMsgRegex = new(@"(\S)*""location"": {(\r|\n){1,2}(.*(\r|\n){1,2}){0,3}(\s)*},");
    private static readonly Regex ClientIp = new(@"["" ""]*http.client_ip: .*,(\r|\n){1,2}");
    private static readonly Regex NetworkClientIp = new(@"["" ""]*network.client.ip: .*,(\r|\n){1,2}");
    private static readonly Regex HashRegex = new(@"(\S)*""hash"": (-){0,1}([0-9]){1,12},(\r|\n){1,2}      ");
    
    public static VerifySettings AddIastScrubbing(this VerifySettings settings)
    {
        settings.AddRegexScrubber(LocationMsgRegex, string.Empty);
        settings.AddRegexScrubber(ClientIp, string.Empty);
        settings.AddRegexScrubber(NetworkClientIp, string.Empty);
        settings.AddRegexScrubber(HashRegex, string.Empty);
    }
}
```

And call it in your tests:

```csharp
var settings = VerifyHelper.GetSpanVerifierSettings();
settings.AddIastScrubbing(); // 👈 
await VerifyHelper.VerifySpans(spans, settings)
                  .UseFileName(filename)
                  .DisableRequireUniquePrefix();
```

Reply:
Cool. Done!